### ISSUE TITLE
Survey Features

### ISSUE THREAD

**Issue by skyuplam** [impact-learning/BJDW#2]
# A predefined survey for the visitors to fill-up for feedback and gathering data purpose.

- [ ] question & answer type (?)

> Scan Preferred QR Code from Choice of 4 QR Codes (Visitor), Exhibitor Login at Exhibit

- [ ] QR code is designated to the survey of the day with information about about the exhibit (?) 
- [ ] any extra information to gather while sending the feedback back (?) e.g. GPS info, mobile device info, etc.

> View Update on Exhibit Visit and Feedback Given by Others

- [ ] a link to view the update on exhibit (?) who's going to manage the update (?) how does the Update works (?)

> Note: On Screen 2, can add “XX% reached 100% of Baitasi”, “XX% reached 100% of DSL”, and “You have visited XX% of DSL”.

- [ ] it means we have to know the total exhibits hosted by Baitasi and DSL, and there is a mean to know which exhibits the visitor has reached (?)

**Comment by CookiesNCream** [comment]
The question and answer type will be provided tomorrow after having a workshop with PS1.

**Comment by CookiesNCream** [comment]
The QR code is designated to the survey with information about that particular exhibit.

**Comment by CookiesNCream** [comment]
Extra information to gather while providing feedback - yes, geo locations, time stamps, device types, IP address etc. (location data of visitors)

**Comment by CookiesNCream** [comment]
View update on Exhibit and Feedback Given by Others - daily visited summary dynamic update, daily visitor count summary dynamic update, reflected as "XX% reached 100% of Baitasi”, “XX% reached 100% of DSL”, and “You have visited XX% of DSL”.

**Comment by skyuplam** [comment]
> The QR code is designated to the survey with information about that particular exhibit.

So the QR is composed from the URI of survey and exhibit ID (let's assign ID to each exhibit). Will the content of visitor survey be changed w.r.t time? e.g. Day 1 we have survey A, and Day 2 we have survey B, etc. If so, how frequent it will be?

**Comment by CookiesNCream** [comment]
Yes, the total number of exhibits is 120, consisting of 80 (DSL) and 40 (BTS).

**Comment by skyuplam** [comment]
> "XX% reached 100% of Baitasi”, “XX% reached 100% of DSL”, and “You have visited XX% of DSL”

Can you elaborate more about these 3 phrases? I know there are 80 DSL and 40 BTS. So If the visitor has answered 10 DSL and 6 BTS survey, it will show 15% reached BTS and 12.5% reached DSL on the update. what about this `You have visited XX% of DSL`?

**Comment by NGaddes** [comment]
_"Whether the content of visitor survey (also for exhibitor survey) will be changed or not for each day of the BJDW, will be provided tomorrow after having a workshop with PS1."_

No, the survey (which will only be one question) will be the same throughout the time of the event.
